src/app/technical-record/vehicle-summary/vehicle-summary-psv/vehicle-summary-psv.component.html

```diff
@@ -1,4 +1,64 @@
 <dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Speed limiter exempt
```

Comment:
I had the speed limiter applicable to psv & hvg, 
Is that only applicable to psv?